docs/source/getting-started/content/proc_starting-codeready-containers-with-proxy.adoc

```diff
@@ -0,0 +1,20 @@
+[id="starting-codeready-containers-with-proxy_{context}"]
+= Starting {prod} behind proxy
+
+Proxy can be defined as part of environment variable or with `crc config set` command option as following:
+
+```
+$ crc config set http-proxy http://example.proxy.com:<port>
+$ crc config set https-proxy http://example.proxy.com:<port>
+$ crc config set no-proxy '<multiple-no-proxy-entries-with-comma-separated>'
+```
+
+Once proxy is set by environment variable or by config, `crc` commands will able to use it.
```

Comment:
We should avoid the use of "config", this is more along the lines of tech slang. Are we talking about the configuration of CodeReady Containers here? I intend to add a section regarding configuration of CRC, so should we include some docs there about how to configure CRC to work within a proxy environment?

Also, honestly, I'm confused by what "once proxy is set" means -- what is it _about_ a proxy that we're setting? This feels vague.